pages/star-search/index.tsx

```diff
@@ -1,34 +1,60 @@
 import { GetServerSidePropsContext } from "next";
+import { captureException } from "@sentry/nextjs";
 import ProfileLayout from "layouts/profile";
 import { useMediaQuery } from "lib/hooks/useMediaQuery";
 import SEO from "layouts/SEO/SEO";
 import useSupabaseAuth from "lib/hooks/useSupabaseAuth";
 import useSession from "lib/hooks/useSession";
 import { StarSearchChat } from "components/StarSearch/StarSearchChat";
+import { UuidSchema, parseSchema } from "lib/validation-schemas";
+
+const SUGGESTIONS = [
+  {
+    title: "Get information on contributor activity",
+    prompt: "What type of pull requests has @brandonroberts worked on?",
+  },
+  {
+    title: "Identify key contributors",
+    prompt: "Who are the most prevalent contributors to the TypeScript ecosystem?",
+  },
+  {
+    title: "Find contributors based on their work",
+    prompt: "Show me the lottery factor for contributors in the remix-run/react-router project?",
+  },
+  {
+    title: "Find experts",
+    prompt: "Who are the best developers that know Tailwind and are interested in Rust?",
+  },
+];
 
 export async function getServerSideProps(context: GetServerSidePropsContext) {
   const searchParams = new URLSearchParams();
-  let sharedPrompt: string | null = null;
+  let sharedChatId: string | null = null;
 
-  if (context.query.prompt) {
-    sharedPrompt = context.query.prompt as string;
-    searchParams.set("prompt", sharedPrompt);
+  if (context.query.id) {
```

Comment:
One thing I was rethinking @jpmcb since the shared conversation work isn't in production yet is to use `share_id` instead of `id`. This would make it more explicit that it was a shared conversation and leave the door open for `id` to be used for loading conversations via the new history work. Thoughts?

Reply:
I think this is a resonable idea 👍

Reply:
+1 - implemented in the API here: https://github.com/open-sauced/api/pull/880